File: 1720-crawler-log-folder/crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        int output = 0;
        for(int i = 0; i < logs.length; i++){
            if(!logs[i].equals("../") && !logs[i].equals("./"))
                output++;
            else if(logs[i].equals("../") && i != 0 && output > 0)
                output--;
        }
        return output;
    }
}

File: 1720-crawler-log-folder/crawler-log-folder.cs
public class Solution {
    public int MinOperations(string[] logs) {
        int output = 0;
        for(int i = 0; i < logs.Length; i++){
            if(!logs[i].Equals("../") && !logs[i].Equals("./"))
                output++;
            else if(logs[i].Equals("../") && i != 0 && output > 0)
                output--;
        }
        return output;
    }
}